plugins/Highway_Parking_Lane.py

```diff
@@ -67,7 +75,7 @@ def way(self, data, tags, nds):
 
         if (("parking:condition:right" in tags and not "parking:lane:right" in tags and not "parking:lane:both" in tags) or
             ("parking:condition:left" in tags and not "parking:lane:left" in tags and not "parking:lane:both" in tags) or
-            ("parking:condition:both" in tags and not "parking:condition:both" in tags)):
+            ("parking:condition:both" in tags and not "parking:lane:both" in tags and not ("parking:lane:left" in tags and "parking:lane:right" in tags))):
```

Comment:
I am not sure, but the condition should not be :
```suggestion
            ("parking:condition:both" in tags and not "parking:lane:both" in tags and not "parking:lane:left" in tags and not "parking:lane:right" in tags)):
```

Reply:
Hi Frodrigo,
Thanks for the review!

I think either could be good.

- In both cases, `parking:condition:both` without `parking:lane:*` is considered bad
- In both cases, `parking:condition:both` together with `parking:lane:both` is considered good
- In both cases, `parking:condition:both` together with `parking:lane:left` + `parking:lane:right` is considered good

However, in my case, `parking:condition:both` with only `parking:lane:left` OR `parking:lane:right` is considered 'bad' (because one side `parking:lane` is not specified, only `parking:condition`); in your case, the latter is considered good too.

I can live with either of the two options, so you can choose which one you prefer :).

Reply:
ok, keep as is.